cabal.project

```diff
@@ -12,6 +12,18 @@ package cardano-submit-api
   tests: True
   ghc-options: -Wall -Werror -fwarn-redundant-constraints
 
+package cardano-db-sync
+  tests: False
+
+package cardano-shell
+  tests: False
+
+package ouroboros-consensus
+  tests: False
+
+package ouroboros-consensus-cardano
+  tests: False
+
```

Comment:
I haven't done so yet, but I imagine that we'd probably want to disable tests for all packages except those that are a part of `cardano-rest`, right?

Reply:
Maybe look at how i now do it in db-sync: https://github.com/input-output-hk/cardano-db-sync/blob/master/cabal.project#L26-L45

Basically disable **all** and then re-enable just tests in the current repo.